arviz/plots/densityplot.py

```diff
@@ -19,6 +19,7 @@ def plot_density(
     var_names=None,
     filter_vars=None,
     combine_dims=None,
+
```

Comment:
This change should be undone, and it looks like it's part of why black is failing CI for this PR. See https://dev.azure.com/ArviZ/ArviZ/_build/results?buildId=5559&view=logs&j=7f538728-4589-5563-0e87-d0823eb40420&t=af0f9f62-4f87-5492-4555-4ab2d5002559&l=20

Reply:
I have fixed this.